frontend/src/components/Auth/FormFormik.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { Formik, Field, Form, ErrorMessage } from "formik";
+import * as Yup from "yup";
+import styled from "styled-components";
+
+import { MdEmail } from "react-icons/md";
+import { RiLockPasswordFill } from "react-icons/ri";
+
+type FormProps = {
+  isSignUp: boolean;
+};
+
```

Comment:
Move all types declarations to separated files i.e FormProps -> FormFormik.types.ts

Reply:
I think co-locate styles in the component are fine for now. Seems I don’t see a lot of benefit of separating them.

Reply:
That's right for this moment, but when styles will grow, then separating them will be inevitable to not clutter up the components. 
Personally, I would keep things organized.